src/ir/effects.h

```diff
@@ -751,6 +751,10 @@ class EffectAnalyzer {
       // traps when ref is null.
       parent.implicitTrap = true;
     }
+    void visitStringViewAccess(StringViewAccess* curr) {
+      // traps when ref is null.
+      parent.implicitTrap = true;
```

Comment:
`stringview_iter.next` also has the side effect of advancing the iterator. We should probably represent that here somehow.

Reply:
I ended up splitting this into 4 classes because of the differences in argument count and side effects, PTAL